src/engine/renderer/tr_surface.cpp

```diff
@@ -1183,9 +1183,9 @@ static void Tess_SurfaceMD5( md5Surface_t *srf )
 				VectorMA( binormal, *boneWeight, tmp, binormal );
 			}
 
-			VectorNormalize( normal );
-			VectorNormalize( tangent );
-			VectorNormalize( binormal );
+			VectorNormalizeFast( normal );
```

Comment:
Unfortunately I don't have an argument for why precision isn't needed here, I only have an argument for why performance is wanted here. 😅️

Reply:
I don't know if that can help, but as far as I know when this code is used, the GLSL compiler is not used (because it is the fallback when that job can't be done on GPU), so we don't have to care about different GLSL compilers behaving differently.